### PR TITLE
net-misc/mstpd: Multiple spanning tree protocol daemon

### PR DESCRIPTION
**Pull request by jsmolic**
* mstpd is an user-space daemon that implements RSTP and MSTP protocols

### CODE REVIEW

File: net-misc/mstpd/mstpd-0.0.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+SRC_URI="https://github.com/mstpd/mstpd/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	default
+
+	sed -i -e 's:-Werror::g' Makefile.am || die
```
Comment (SoapGentoo) [inline net-misc/mstpd/mstpd-0.0.8.ebuild:19]: can you tell upstream to make this an option? saves us the (slow) autoreconf

File: net-misc/mstpd/mstpd-0.0.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		--localstatedir="${EPREFIX}"/var
+}
+
+src_install() {
+	default
+
+	# install systemd service
+	systemd_dounit "${FILESDIR}"/${PN}.service
+
+	docompress -x /usr/share/man/man{5,8}
```
Comment (SoapGentoo) [inline net-misc/mstpd/mstpd-0.0.8.ebuild:35]: is the compression a problem?

Comment (jsmolic) [reply net-misc/mstpd/mstpd-0.0.8.ebuild:35]: It's just a QA warning

File: net-misc/mstpd/mstpd-0.0.8.ebuild (hunk 1 above)
Comment (jsmolic) [reply net-misc/mstpd/mstpd-0.0.8.ebuild:19]: Sure, I'll forward it

Comment (jsmolic) [reply net-misc/mstpd/mstpd-0.0.8.ebuild:19]: Forwarded in https://github.com/mstpd/mstpd/issues/105

**Pull request closed without merging**